backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/ApiUserManagementTest.java

```diff
@@ -781,6 +781,69 @@ void getUsers_orgUser_failure() {
     runQuery("users-query", "Current user does not have permission to request [/users]");
   }
 
+  @Test
+  void getUser_adminUser_success() {
+    useOrgAdmin();
+
+    // necessary to get user id
+    List<ObjectNode> usersRetrieved = fetchUserList();
+    assertTrue(usersRetrieved.size() >= 2);
+
+    ObjectNode nobodyUserFromList =
+        usersRetrieved.stream()
+            .filter(node -> "nobody@example.com".equals(node.get("email").asText()))
+            .findAny()
+            .get();
+
+    String nobodyUserId = nobodyUserFromList.get("id").asText();
+
+    // this is the query being tested
+    JsonNode userRetrieved = runQuery("user-query", getGetUserVariables(nobodyUserId)).get("user");
+
+    assertEquals(
+        nobodyUserFromList.get("firstName").asText(), userRetrieved.get("firstName").asText());
+    assertEquals(nobodyUserFromList.get("email").asText(), userRetrieved.get("email").asText());
+    assertEquals("DIS_ORG", userRetrieved.get("organization").get("externalId").asText());
+    assertEquals(
+        EnumSet.of(
+            UserPermission.UPDATE_TEST,
+            UserPermission.START_TEST,
+            UserPermission.SEARCH_PATIENTS,
+            UserPermission.SUBMIT_TEST),
+        extractPermissionsFromUser((ObjectNode) userRetrieved));
+  }
+
+  @Test
+  void getUser_adminUserWrongOrg_failure() {
+    useOrgAdmin();
+
+    // necessary to get user id
+    List<ObjectNode> usersRetrieved = fetchUserList();
+    assertTrue(usersRetrieved.size() >= 2);
+
+    ObjectNode nobodyUserFromList =
+        usersRetrieved.stream()
+            .filter(node -> "nobody@example.com".equals(node.get("email").asText()))
+            .findAny()
+            .get();
+
+    String nobodyUserId = nobodyUserFromList.get("id").asText();
+
+    // user has permission to make the query, but not to access the user
+    useOutsideOrgAdmin();
+    runQuery("user-query", getGetUserVariables(nobodyUserId), "Cannot determine user's identity.");
```

Comment:
this is really a comment on the main logic itself, not the test -- this should be complaining about permissions restrictions first and foremost. hopefully switching the permission annotation will fix that.